aws/resource_aws_lb_listener_certificate.go

```diff
@@ -66,24 +75,26 @@ func resourceAwsLbListenerCertificateCreate(d *schema.ResourceData, meta interfa
 	}
 
 	if err != nil {
-		return fmt.Errorf("error adding LB Listener Certificate: %s", err)
+		return fmt.Errorf("error adding LB Listener Certificate: %w", err)
 	}
 
-	d.SetId(d.Get("listener_arn").(string) + "_" + d.Get("certificate_arn").(string))
+	d.SetId(fmt.Sprintf("%s_%s", listnerArn, certificateArn))
```

Comment:
```suggestion
	d.SetId(tfelbv2.ListenerCertificateCreateID(listenerArn, certificateArn))
```

and add `ListenerCertificateCreateID()`.